test/unit/rules/invocable-blacklist-test.js

```diff
@@ -5,7 +5,7 @@ const generateRuleTests = require('../../helpers/rule-test-harness');
 generateRuleTests({
   name: 'invocable-blacklist',
 
-  config: ['foo', 'bar'],
+  config: ['error', ['foo', 'bar']],
```

Comment:
This test should not need to be changed (this indicates that this is a breaking change...) 😸

Reply:
👍